### PR TITLE
Clarify scope of End User License Agreement (EULA)

### PR DESCRIPTION
**Pull request by antoineco**
This PR adds a note to the README which clarifies what software artifacts the included EULA applies to.

Feel free to suggest a different wording.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,19 @@ details.
 
 Although this project is not part of the [CNCF][cncf], we abide by its [code of conduct][cncf-conduct].
 
+## License
+
+This software is licensed under the [Apache License, Version 2.0][asl2].
+
+Additionally, the End User License Agreement included in the [`EULA.pdf`](./EULA.pdf) file applies to software artifacts
+released by TriggerMesh Inc.: compiled executables, container images, etc.
```
Comment (antoineco) [inline README.md:44]: Note: I wasn't sure about the "etc." here. Should we keep the list exhaustive instead, so it's not left to interpretation?

Comment (sebgoa) [reply README.md:44]: I think we can remove the `etc`

Comment (sebgoa) [inline README.md:44]:
```suggestion
released by TriggerMesh Inc.: compiled executables and container images.
```

**Review by sebgoa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (antoineco) [reply README.md]: Applied in [ebe1f00..6d0ff0f2](https://github.com/triggermesh/triggermesh/compare/ebe1f00..6d0ff0f2)

**Comment by antoineco** [comment]
(It's marked as "Closed" but was effectively merged. Looks like a GitHub bug. Please ignore.)

**Pull request closed without merging**